.github/workflows/build_and_deploy.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     tags:
       - 'v*.*.*'
+      - '!v*.*.*-beta'
```

Comment:
Ignoring any tags created that end in `-beta` since these will be run from another Workflow